content/_partials/toptoolbar.html.haml

```diff
@@ -60,6 +60,19 @@
             %a.dropdown-item.feature{:href => expand_link(page.url)}
               = '&nbsp;-&nbsp;' + page.title
 
+      %li.dropdown.nav-item
+        = partial("dropdown.html.haml", :name => "Member of CDF")
+
+        .dropdown-menu
+          %a.dropdown-item.feature{:href => "https://cd.foundation/"}
```

Comment:
```suggestion
          %a.dropdown-item.feature{:href => "https://cd.foundation/about/"}
```

Given the label, linking directly to this page makes more sense.

Reply:
Logically, you are correct. 

But unfortunately the CDF About page consists of paragraphs of text. I'd rather lead users to the CDF's colorful homepage, with a succinct summary of the CDF's purpose and clear links to more information, than a wall of text. That way they instantly get a quick summary of the project in the most accessible and appealing way and can decide if they want more information.